site/docs/master/style-guide.md

```diff
@@ -0,0 +1,313 @@
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>In version 1.4, ...</td><td>In the current version, ...</td></tr>
+<tr><td>The Federation feature provides ...</td><td>The new Federation feature provides ...</td></tr>
+</table>
+
+### Language
+
+This documentation uses U.S. English spelling and grammar.
+
+## Documentation formatting standards
+
+
+### Use camel case for API objects
+
+When you refer to an API object, use the same uppercase and lowercase letters
+that are used in the actual object name. Typically, the names of API
+objects use
+[camel case](https://en.wikipedia.org/wiki/Camel_case).
+
+Don't split the API object name into separate words. For example, use
+PodTemplateList, not Pod Template List.
+
+Refer to API objects without saying "object," unless omitting "object"
+leads to an awkward sentence construction.
+
+<table caption="Do and Don't - API objects" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>The Pod has two containers.</td><td>The pod has two containers.</td></tr>
+<tr><td>The Deployment is responsible for ...</td><td>The Deployment object is responsible for ...</td></tr>
+<tr><td>A PodList is a list of Pods.</td><td>A Pod List is a list of pods.</td></tr>
+<tr><td>The two ContainerPorts ...</td><td>The two ContainerPort objects ...</td></tr>
+<tr><td>The two ContainerStateTerminated objects ...</td><td>The two ContainerStateTerminateds ...</td></tr>
+</table>
+
+### Use angle brackets for placeholders
+
+Use angle brackets for placeholders. Tell the reader what a placeholder
+represents.
+
+1. Display information about a Pod:
+
+        kubectl describe pod <pod-name> -n <namespace>
+
+    If the namespace of the pod is `default`, you can omit the '-n' parameter.
+
+### Use bold for user interface elements
+
+<table caption="Do and Don't - Bold interface elements" markdown="1">
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td markdown="1">Click **Fork**.</td><td>Click "Fork".</td></tr>
+<tr><td markdown="1">Select **Other**.</td><td>Select "Other".</td></tr>
+</table>
+
+### Use italics to define or introduce new terms
+
+<table caption="Do and Don't - Use italics for new terms" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>A _cluster_ is a set of nodes ...</td><td>A "cluster" is a set of nodes ...</td></tr>
+<tr><td>These components form the _control plane_.</td><td>These components form the **control plane**.</td></tr>
+</table>
+
+### Use code style for filenames, directories, paths, object field names and namespaces
+
+<table caption="Do and Don't - Use code style for filenames, directories, paths, object field names and namespaces" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>Open the `envars.yaml` file.</td><td>Open the envars.yaml file.</td></tr>
+<tr><td>Go to the `/docs/tutorials` directory.</td><td>Go to the /docs/tutorials directory.</td></tr>
+<tr><td>Open the `/_data/concepts.yaml` file.</td><td>Open the /\_data/concepts.yaml file.</td></tr>
+</table>
+
+### Use punctuation inside quotes
```

Comment:
How about parenthesis? Ponctuation should go inside or outside? I'm just taking this opportunity to ask :) we might not need to add this to the guide.

Reply:
For parentheses, punctuation should go outside the ending ), unless the statement inside the () is a full sentence. But also, i think putting things in () can get a little confusing, or skipped entirely by a reader. So for docs, id say try to decrease the amount of use, and consider writing in a way that avoids ()